src/components/profile/update.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+class UserUpdate extends React.Component {
+  constructor(props) {
```

Comment:
just for a readiblity thing, can you name the file the same as the component, ie this file is UserUpdate.js?